ui/lib/pki/addon/routes/issuers/issuer/details.js

```diff
@@ -1,3 +1,23 @@
-import Route from '@ember/routing/route';
+import PkiIssuersIndexRoute from '../index';
 
-export default class PkiIssuerDetailsRoute extends Route {}
+export default class PkiIssuerDetailsRoute extends PkiIssuersIndexRoute {
```

Comment:
I noticed that for keys we fetch the model in `routes/keys/key.js` so we don't have to subsequently fetch in both `routes/keys/key/edit.js` and `routes/keys/key/details.js` 

Do you prefer to do it this way, directly in the `details.js` route? I can refactor keys if we want to be consistent!

Reply:
I like that pattern! Issuers doesn't have a `routes/issuers/issuer.js` route yet so I'll add that